parser.py

```diff
@@ -510,13 +250,16 @@ def uop(self, *args):
             # <id> <anylist> , <uop>
             # <id>() <anylist>, <uop>
             # <id>(<explist>) <anylist>, <uop>
-            op = Op(op_name, params, wires)
+            # This case only appears for tensor products of operators
+            # TODO: consider extending the grammar so users can specify
+            #  "simultaneous" gates/channels via a tensor product
```

Comment:
I wouldn't mind keeping simultaneous gates/channels as they are, e.g.,
```c
z q[2];
s q[2];
h q[2];
```

Using the tensor product as well here add complexity, since there is now two different ways of specifying the same thing:

```c
z q[2] @ s q[2] @ h q[2];
```

(assuming that's what you mean by simultaneous gate applications).

Reply:
P.s. we still need to get around to implementing https://github.com/co9olguy/qdata/commit/e5b89849c17c96705ee3888e108104afcafef0b2#r42899786. This is a almost trivial change in the parser (`,`->`@`), but I find this a _massive_ readability improvement over CSV

Reply:
I asked people what they thought about this. Nobody felt as strongly as you do :laughing:

Reply:
Simultaneous gates should be applied to different wires, no? The idea of simultaneous gates is you are deliberately asking the executor to run those gates at the same time, whereas if you didn't, it would be up to the compiler to schedule

Reply:
OH! I get it. Okay, I'm now in favour, that is pretty cool.